source/OptimaJet.DWKit.StarterApplication/Utility/ApplicationExtensions.cs

```diff
@@ -38,6 +39,11 @@ public static IApplicationBuilder UseWorkflow(this IApplicationBuilder app, ICon
 
         public static IApplicationBuilder UseBuildEngine(this IApplicationBuilder app, IConfigurationRoot configuration)
         {
+            var sampleDataApiToken = GetVarOrDefault("SAMPLEDATA_BUILDENGINE_API_ACCESS_TOKEN", String.Empty);
```

Comment:
I just mean that this behavior doesn't seem tied to sample data, but more just a default / override.

like, for production, we could have the exact same code, provided we use the phrasing:

```cs
var defaultToken = GetVarOrDefault("DEFAULT_BUILDENGINE_API_ACCESS_TOKEN", String.Empty);

if (!String.IsNullOrEmpty(defaultToken))
{
    // somewhere in the BuildEngine Code, a method should be abstracted for accessing the API token
    // which would either get the set organization token, or the default token from here.
    // and, actually, if the BuildEngine stuff is somewhat stateless w/r/t the api token acces, 
    // these code changes could be remove entirely, and we could just do the abstracted method for api token 
    // access to either be the org specified token or the DEFAULT_BUILDENGINE_API_ACCESS_TOKEN
    BackgroundJob.Enqueue<BuildEngineSystemMonitor>(service => service.SetApiToken(defaultToken));
}
```

Reply:
how do you imagine it work in production?

Reply:
For BuildEngine, we have an admin interface for creating them.  I believe they were supposed to be filled in by the SuperAdmin in the SuperAdmin UI.

Reply:
ah, so you just don't want a default in production? (this is fine, I'm just trying to figure out what the intent is)

Reply:
"For CI and Dev environments, we bootstrap regularly and need to have Organization.BuildEngineApiAccessToken updated."

Reply:
right, but that just felt narrow to me, leaving production out of the picture. Apologies if I wasn't clear in expressing my confusion.

If the intent is to:

 - never have a default build engine configuration in production
 - it is up to the super admin to enable the default build engine via manual configuration

that's fine, I'd just like to know if that _is_ the case

Reply:
Yep.  This is just for Dev and CI. Not for production. For Production, we already have this exposed through the Organization UI.  I forgot we had already done this.  No need for SuperAdmin UI.

Reply:
aight. this PR is :+1: then. thanks for explaining!